test/RadworksDripsGovernance.t.sol

```diff
@@ -2,8 +2,25 @@
 pragma solidity ^0.8.20;
 
 import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
+import {UUPSUpgradeable} from "@openzeppelin/contracts/proxy/utils/UUPSUpgradeable.sol";
 import {ProposalTest} from "test/helpers/ProposalTest.sol";
 
+import {console2} from "forge-std/console2.sol";
+
+/// @dev This contract used in the testing of using governance to upgrade the Drips protocol
```

Comment:
"*is* used in the testing of *governance* to upgrade the Drips protocol*.*" this sounds a little better

Reply:
Done: https://github.com/ScopeLift/radworks-governor-upgrade/pull/21/commits/79efaeab8b3dbcbd6b2bb47c70c5ffd7cbe123ed